doc/classes/AnimationNodeStateMachinePlayback.xml

```diff
@@ -93,4 +93,18 @@
 	<members>
 		<member name="resource_local_to_scene" type="bool" setter="set_local_to_scene" getter="is_local_to_scene" overrides="Resource" default="true" />
 	</members>
+	<signals>
+		<signal name="state_finished">
+			<param index="0" name="state" type="StringName" />
+			<description>
+				Emitted when the [param state] finishes playback. If [param state] is a state machine set to grouped mode, its signals are passed through with its name prefixed.
```

Comment:
```suggestion
				Emitted when the [param state] finishes playback. If [param state] is a state machine set to grouped mode, its signals are passed through with its name prefixed.
				If there is a crossfade, this will be fired when the influence of the [method get_fading_from_node] animation is no longer present.
```